src/kubectlUtils.ts

```diff
@@ -130,6 +130,22 @@ export async function getGlobalResources(kubectl: Kubectl, resource: string): Pr
     });
 }
 
+export async function getCRDTypes(kubectl: Kubectl): Promise<CRD[]> {
+    const crdTypes = await kubectl.asJson<KubernetesCollection<any>>(`get crds -o json`);
+    if (failed(crdTypes)) {
+        vscode.window.showErrorMessage(crdTypes.error[0]);
+        return [];
+    }
+
+    return crdTypes.result.items.map((item) => {
+        return {
+            metadata: item.metadata,
+            kind: item.spec.names.kind,

```

Comment:
I don't think this is right.  The `kind` of a Custom Resource _Definition _is `CustomResourceDefinition`; this seems to confuse the kind of the _definition_ with the kind of an _instance_.  I don't think we use this, though, so I think maybe we should just merge the PR to unblock you, and tweak later.